src/components/RangeSlider/CustomLabel.js

```diff
@@ -57,7 +57,11 @@ class CustomLabel extends Component {
       oneMarkerValue,
       oneMarkerLeftPosition,
       oneMarkerPressed,
+      roundLabelValue
     } = this.props
+
+    const roundedValue = roundLabelValue(oneMarkerValue)
```

Comment:
@tobyjsullivan This would definitely be a better approach. But it seems the issue is that oneMarkerValue is provided by the parent component MultiSlider that is an external package. So we don't have access to it in RangeSlider. Do you know if there's another way to go about getting that variable before it's passed to CustomLabel?

Reply:
Yeah, all the props are passed to `CustomLabel` in `RangeSlider/index.js`.

https://github.com/AdaloHQ/slider-component/blob/7e0a0b1fe17e20ea1536e28e14f90f1065cf9d3b/src/components/RangeSlider/index.js#L125-L139

That suggests `oneMarkerValue` is being passed as part of the props spreading (the part that looks like `{...props}`).

That means it should be possible to intercept that prop and round it there.

```js
            customLabel={props => {
              let { oneMarkerValue } = props
              oneMarkerValue = this.roundLabelValue(oneMarkerValue)

              return (
                <CustomLabel
                  {...props}
                  oneMarkerValue={oneMarkerValue}
                  bgColor={bgColor}
                  txtColor={txtColor}
                  font={font}
                  labelRounding={labelRounding}
                  bodyFont={
                    track.styles
                      ? track.styles.bodyFont
                      : { fontFamily: _fonts.body }
                  }
                />
              )
            }}
```

Reply:
This makes sense--implemented!! Thank you @tobyjsullivan!